03-version-control-with-github.Rmd

```diff
@@ -7,29 +7,34 @@ ottrpal::set_knitr_image_path()
 
 ## Learning Objectives
 
+AVOCADO: 1. Following the instructions, I sometimes got confused about when to use GitHub and when to use GitKraken. It was mentioned on the document, so I believe people could locate/understand it, but maybe it could be clearer if it could be stated in each section/paragraph or have direct cues such as “Let’s move on to GitKraken to clone the local repository”. “We can use GitKraken to perform these tasks ABC.”
+AVOCADO: 2. I think the highlights with change in font (coding font?) really makes it clearer. It might be helpful to also highlight, or just add a different color, (e.g. blue) to underscore terminologies like pull requests when they appear in the course for the first time.
```

Comment:
Great idea! I'll do that too, or make an info box or something!

Reply:
oh shoot I still need to add my different style boxes to the template - I can do that tomorrow!